parcellab/app/templates/deployment.yaml

```diff
@@ -5,9 +5,11 @@ metadata:
   name: {{ $fullname }}
   labels:
     {{- include "common.labels" . | nindent 4 }}
+    {{- if and .Values.datadog .Values.datadog.enabled }}
     tags.datadoghq.com/env: {{ include "common.env" . }}
     tags.datadoghq.com/service: {{ $fullname }}
     tags.datadoghq.com/version: {{ include "common.version" . }}
+    {{- end }}
```

Comment:
Actually these labels might not be needed at all, the ones from the spec are the only ones that will be applied to the pods, isn't it?

Reply:
Yes, according to the docs it is not needed at deployment level

Reply:
removed